src/components/Author/index.tsx

```diff
@@ -45,6 +46,7 @@ function Author({ image, username, date, favorites, isFavorited, slug }: Props)
           <div>
             <span className={styles.username}>{username}</span>
             <span>{date}</span>
+            <span data-testid="isUserFollow">{following && '*** You follow this user'}</span>
```

Comment:
if the boolean is false, you will be going to render empty spam. maybe you can do it. and add the text to i18next.
```suggestion
            {following && (
              <span data-testid="isUserFollow">{'*** You follow this user'}</span>
             )}
```